python/turbodbc/cursor.py

```diff
@@ -7,23 +7,24 @@
 
 from .exceptions import translate_exceptions, InterfaceError, Error
 
-
 def _has_numpy_support():
+
     try:
         import turbodbc_numpy_support
         return True
     except ImportError:
-        return False
+         return False
 
-
-def _make_masked_array(data, mask):
+def _make_masked_arrays(result_batch):
     from numpy.ma import MaskedArray
     from numpy import object_
-    if isinstance(data, list):
-        return MaskedArray(data=data, mask=mask, dtype=object_)
-    else:
-        return MaskedArray(data=data, mask=mask)
-
+    numpy_masked_array = []
```

Comment:
please use the plural form `numpy_masked_arrays` (or even leave the `numpy` part out)